docs/configuration.md

```diff
@@ -0,0 +1,62 @@
+# Configuration
+
+REMS contains a number of configuration options that can be used to alter authentication options, theming or to add integration points, just to name a few.
+
+Configuration can be found under the `env` folder and [cprop](https://github.com/tolitius/cprop) library is used to handle the configuration options. For example, to change configuration for your production environment edit the `config.edn` file under the `env/prod/resources` folder.
+
+## Authentication options
+
+Currently supported authentication methods are SAML2 and LDAP. Login method to be used can be defined with the key `:authentication` and the following values are recognized:
+
+* `:fake-shibboleth`
+* `:shibboleth`
+* `:ldap`
+
+### SAML2
+
+When using this option, login requests are directed to `/Shibboleth.sso/Login`.
+
+### LDAP
+
+Using LDAP as the authentication method requires that additional configuration with the following structure is provided:
+```
+:ldap {:connection {:host "your-host-name"
+                    :ssl? true}
+       :search-root "dc=some,dc=thing"}
+```
+
+### Development login
```

Comment:
Here also configuration option not mentioned in heading so unclear what it's talking about